web/ui/react-app/src/App.css

```diff
@@ -135,16 +135,42 @@ div.time-input {
 }
 
 .graph-legend {
-    margin: 15px 0 15px 25px;
-    font-size: 0.8em;
+    margin: 15px 0 15px 55px;
+    font-size: 0.75em;
+    background: #222;
+    color: white;
+    padding: 10px 5px;
+    display: inline-block;
+    cursor: pointer;
+}
+
+.legend-item {
+    display: flex;
+    align-items: center;
+    padding: 0 5px;
+    border-radius: 3px;
+    user-select: none;
```

Comment:
Can we still allow user-select? It was sometimes useful in the old UI when you want to copy a time series name.

Reply:
Yes, sure. I will remove it.